crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -24,6 +24,14 @@ struct FragmentInput {
 #endif
 };
 
+fn screen_space_dither(frag_coord: vec2<f32>) -> vec4<f32> {
+	// Iestyn's RGB dither (7 asm instructions) from Portal 2 X360, slightly modified for VR.
+    var dither = vec3<f32>(dot(vec2<f32>(171.0, 231.0), frag_coord));
+    dither = fract(dither.rgb / vec3<f32>(103.0, 71.0, 97.0));
+	// Subtract 0.5 to avoid slightly brightening the whole viewport.
+    return vec4<f32>((dither.rgb - 0.5) / 255.0, 0.0);
+}
```

Comment:
Where did this code originate from? Did you modify it for VR yourself, or did someone else do so?

Reply:
Another reason this is marked as draft - I haven't looked at the licenses for any of my references. Might need some support for that. Unsure if little code snippets like this are acceptable.